### PR TITLE
Remove detectors

### PR DESCRIPTION
**Pull request by ldelossa**
Closes milestone: https://github.com/coreos/clair/issues/745

Removes detectors from AncestryGet rpc call. Response is now:

```
{
  "ancestry": {
    "name": "test-ancestry",
    "layers": [
      {
        "layer": {
          "hash": "8e674ad76dcef6f6d0398bc25550f680f8751876064a87a15347f00687492090"
        }
      }
    ]
  }
}
```

### CODE REVIEW

File: api/v3/clairpb/clair.proto [hunk 1]
```diff
@@ -155,16 +155,11 @@ message GetAncestryResponse {
...
-        // The detectors used to scan this Ancestry. It may not be the current
-        // set of detectors in clair status.
-        repeated Detector detectors = 2;
         // The list of layers along with detected features in each.
         repeated AncestryLayer layers = 3;
     }
     // The ancestry requested.
     Ancestry ancestry = 1;
-    // The status of Clair at the time of the request.
-    ClairStatus status = 2;
```
Comment (jzelinskie) [inline api/v3/clairpb/clair.proto]: We still need the status so that clients can determine when they need to rescan something.

**Review by jzelinskie — CHANGES_REQUESTED** [review]

File: api/v3/clairpb/clair.proto (hunk 1 above)
Comment (ldelossa) [reply api/v3/clairpb/clair.proto]: :+1:

**Comment by ldelossa** [comment]
@jzelinskie 

```{
  "ancestry": {
    "name": "test-ancestry",
    "layers": [
      {
        "layer": {
          "hash": "8e674ad76dcef6f6d0398bc25550f680f8751876064a87a15347f00687492090"
        }
      }
    ]
  },
  "status": {
    "detectors": [
      {
        "name": "apk",
        "version": "1.0",
        "dtype": "DETECTOR_D_TYPE_FEATURE"
      },
      {
        "name": "dpkg",
        "version": "1.0",
        "dtype": "DETECTOR_D_TYPE_FEATURE"
      },
      {
        "name": "rpm",
        "version": "1.0",
        "dtype": "DETECTOR_D_TYPE_FEATURE"
      },
      {
        "name": "apt-sources",
        "version": "1.0",
        "dtype": "DETECTOR_D_TYPE_NAMESPACE"
      },
      {
        "name": "lsb-release",
        "version": "1.0",
        "dtype": "DETECTOR_D_TYPE_NAMESPACE"
      },
      {
        "name": "os-release",
        "version": "1.0",
        "dtype": "DETECTOR_D_TYPE_NAMESPACE"
      },
      {
        "name": "redhat-release",
        "version": "1.0",
        "dtype": "DETECTOR_D_TYPE_NAMESPACE"
      },
      {
        "name": "alpine-release",
        "version": "1.0",
        "dtype": "DETECTOR_D_TYPE_NAMESPACE"
      }
    ],
    "last_update_time": "2019-07-03T21:41:56.000Z"
  }
}```

updated api output.

**Comment by jzelinskie** [comment]
LGTM if you squash the last two commits into one.

**Comment by ldelossa** [comment]
@jzelinskie squashed.

**Comment by ldelossa** [comment]
@jzelinskie should be good to go. Added .clang-format, make targets for proto linting and jsonnet compilation. CI passing.

File: Makefile [hunk 2]
```diff
@@ -49,3 +49,12 @@ teardown-local:
...
 	-helm delete --purge clair-pg
+
+.PHONY: lint-proto
+lint-proto:
+	clang-format -i api/v3/clairpb/clair.proto
+
+
+.PHONY: gen-drone-config
+gen-drone-config:
+	jsonnet -y .drone.jsonnet -o .drone.yml
```
Comment (jzelinskie) [inline Makefile]: I think we should make this `drone jsonnet --stream` since that's what their documentation says to use.

**Review by jzelinskie — CHANGES_REQUESTED** [review]
1 final change

**Review by jzelinskie — APPROVED** [review]

**Pull request merged**